index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link href="styles/normalize.css" rel="stylesheet">
+  <link href="styles/style.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:100,300,400" rel="stylesheet">
+</head>
+<body>
+  <div class="calculator">
+    <div class="calculator-answer-screen">0</div>
+    <div class="calculator-row">
+      <div class="calculator-cell calculator-AC">AC</div>
```

Comment:
> Is it just because the purpose of a button is to be clicked so why have a div do a button's work?

Pretty much. you can technically make a `<div>` act almost exactly like a `<button>` but it takes some extra configuration. Some of which is 

- you'll need to add `cursor: pointer` to your css
- If you want to be able to tab through these faux buttons you'll also need to add a `tabIndex` attribute to each div
- i sure there is some other small things here 


I'm glad you're questioning things. Its healthy. But to flip it back to you, you should be able to defend why you're using divs here instead of buttons.